sunpy/map/mapbase.py

```diff
@@ -672,8 +672,12 @@ def spatial_units(self):
         """
         Image coordinate units along the x and y axes (i.e. cunit1, cunit2).
         """
-        return SpatialPair(u.Unit(self.meta.get('cunit1', 'arcsec')),
-                           u.Unit(self.meta.get('cunit2', 'arcsec')))
+        cunit1 = self.meta.get('cunit1')
+        cunit2 = self.meta.get('cunit2')
+        if (cunit1 is None) or (cunit2 is None):
+            raise AttributeError('Image coordinate units not present in metadata')
```

Comment:
Should we have an argument to force the use of a particular unit? Or  a helpful message on how to create a map from this type of data?

Reply:
I could add a little bit more to this message that says how to set the unit metadata of the map.

Reply:
How about now?